src/feature-selector-form/feature-selector-form.tsx

```diff
@@ -35,6 +49,16 @@ export function FeatureSelectorForm( { onContinue }: Props ) {
 
 	const selectedFeatureId = useAppSelector( selectSelectedFeatureId );
 
+	const localRelevantTaskIds = useMemo( () => {
+		if ( selectedFeatureId ) {
+			return getRelevantTaskIds(
+				{ ...issueDetails, featureId: selectedFeatureId },
+				reportingConfig
+			);
+		}
+		return [];
+	}, [ selectedFeatureId, issueDetails, reportingConfig ] );
```

Comment:
**note:** Adding a comment so I can make a reference to p1683110760824139-slack-CQD1HH4MA